source/auth/auth.rst

```diff
@@ -106,6 +108,8 @@ Errors
 
 Drivers SHOULD raise an error as early as possible when detecting invalid values in a credential. For instance, if a ``mechanism_property`` is specified for `MONGODB-CR`_, the driver should raise an error indicating that the property does not apply.
 
+Drivers MUST raise an error if any required information for a mechanism is missing. For instance, if a ``username`` is not specified for `SCRAM-SHA-256`, the driver must raise an error indicating the the property is missing.
```

Comment:
Double backticks for SCRAM-SHA-256 or omit them entirely :)

Reply:
I saw that backticks were used in the previous line with `MONGODB_CR`, but I have removed the backticks for SCRAM-SHA-256.